Source/WTF/wtf/threads/Signals.h

```diff
@@ -174,16 +175,93 @@ using WTF::SignalHandler;
 using WTF::addSignalHandler;
 using WTF::activateSignalHandlersFor;
 
-#else // not OS(UNIX)
+#elif OS(WINDOWS) // not OS(UNIX)
```

Comment:
If there is no `OS` that is neither `UNIX` or `WINDOWS` then this can just be an `#else`.